.github/workflows/pr.yaml

```diff
@@ -37,36 +37,44 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
         with:
           fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Check Image and Relevant Changes
         id: check-image
         uses: ./.github/actions/check-image-and-changes
 
       - name: Check if build should be skipped
         run: |
+          if [[ "${{ steps.check-image.outputs.is_skipped }}" == "true" ]]; then
+            echo "::notice::Skipping build: [skip-build] tag found in commit messages."
+          fi
           if [[ "${{ steps.check-image.outputs.relevant_changes }}" == "false" && "${{ steps.check-image.outputs.image_exists }}" == "true" ]]; then
```

Comment:
where below this line do you ACTUALLY check for these `check-image` env vars? Are you only testing if `is_skipped` is true?